### PR TITLE
Check broker address status

### PR DESCRIPTION
**Pull request by lulf**
Less risky version.

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=shared

**Comment by lulf** [comment]
@enmasse-ci run tests profile=isolated

**Comment by lulf** [comment]
@enmasse-ci run tests profile=shared

**Comment by lulf** [comment]
@enmasse-ci run tests profile=shared

File: standard-controller/src/main/java/io/enmasse/controller/standard/AddressController.java [hunk 1]
```diff
@@ -745,30 +745,76 @@ private void checkAndRemoveDrainingBrokers(Set<Address> addresses) throws Except
...
                     ok += RouterStatus.checkForwarderLinks(address, routerStatusList);
                     break;
                 case "topic":
-                    ok += checkBrokerStatus(address, clusterOk);
+                    ok += checkBrokerStatus(brokerStatusCollector, address, clusterOk, clusterAddresses, clusterQueues);
                     for (RouterStatus routerStatus : routerStatusList) {
                         ok += routerStatus.checkLinkRoutes(address);
                     }
                     if (isPooled(addressPlan)) {
+                        for (BrokerStatus brokerStatus : address.getStatus().getBrokerStatuses()) {
```
Comment (k-wall) [inline standard-controller/src/main/java/io/enmasse/controller/standard/AddressController.java]: Why is this check only appropriate on the isPooled branch?

Comment (lulf) [reply standard-controller/src/main/java/io/enmasse/controller/standard/AddressController.java]: For sharded topics, the addresses are part of the static broker.xml.

Comment (k-wall) [reply standard-controller/src/main/java/io/enmasse/controller/standard/AddressController.java]: > For sharded topics, the addresses are part of the static broker.xml.

I'd forgotten that difference.  Thanks.

**Comment by lulf** [comment]
@enmasse-ci run tests profile=shared

**Pull request merged**